src/base/static/components/organisms/featured-places-navigator.tsx

```diff
@@ -72,55 +84,33 @@ const FeaturedPlacesNavigator = props => {
             place,
             iconUrl:
               featuredPlace.iconUrl ||
-              (
-                props.placeConfig.place_detail.find(
-                  config => config.category === (place || {}).location_type,
-                ) || {}
-              ).icon_url,
+              ((props.placeConfig.place_detail.find(
+                config =>
+                  config.category === ((place || {}) as Place).location_type,
+              ) || {}) as any).icon_url,
```

Comment:
Note that casting to `any` isn't ideal, but I figure it's okay for now since I'm just porting existing code and couldn't figure out how to type this.